### DISCUSSION TITLE
Dreambooth?

### DISCUSSION THREAD

**Discussion by githubUser01946** [vladmandic/sdnext#1534] [Q&A]
I know this is not an `Issue` so I am posting here because I am probably missing something. Like the brain capacity.

Anyway, I have installed Dreambooth., more specifically I have installed `[sd_dreambooth_extension](https://github.com/d8ahazard/sd_dreambooth_extension)` and there should be a tab that says Dreambooth but there isn't. There is a tab that says Train but that was there before, where is the Dreambooth section hiding?

Under the title of the extension it does say the following `tab, training, installed` but I cannot see it anywhere. My tabs can be seen in the image below.

![image](https://github.com/vladmandic/automatic/assets/117453319/63eb1745-d764-4f18-94ab-496356ac3105)

**Comment by vladmandic** [comment]
Check the console logs if extension loaded/initialized correctly.

**Reply by githubUser01946** [reply]
Hello, I had to do something with a Discord thing for it to work. It shows now.
